### PR TITLE
Cleaning data with spreadsheets wiki

### PR DESCRIPTION
**Pull request by jwoolbright23**

### CODE REVIEW

File: Cleaning-Spreadsheets.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Cleaning Data with Spreadsheets
+
+## In the prep work for this class, the students learned:
+
+1. What Cleaning data is.
```
Comment (gildedgardenia) [inline Cleaning-Spreadsheets.md]:
```suggestion
1. What cleaning data is.
```

File: Cleaning-Spreadsheets.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+### Best Practices for ALL Studios
+1. Check in with EVERY student during EVERY studio.
+   1. This builds relationships and trust, and it helps you spot potential issues before they get too large.
+   1. Checking in does NOT mean a quick question like, "How are you doing? Need any help?" Instead, sit next to the student and ask specific questions about their code, their interpretation of the instructions, or their coding dreams.
+   1. Provide detailed feedback as often as possible. "Good job!" does not cut it. An example would be "Hey! I like your variable names! They are very descriptive!".
+1. Be prepared to clarify the studio instructions beyond just re-reading the words on the screen.
+1. Encourage students to work together and share ideas.
+1. Assist individuals as questions arise. Address frequent mistakes and/or questions to your whole group.
+1. Make a note of any issues that occur during the studio and provide that feedback to the instructor and LaunchCode team.
```
Comment (gildedgardenia) [inline Cleaning-Spreadsheets.md]: I think we only really need this section on the first day of class wiki.

**Review by gildedgardenia — APPROVED** [review]
Looks good!

File: Cleaning-Spreadsheets.md (hunk 2 above)
Comment (jwoolbright23) [reply Cleaning-Spreadsheets.md]: Removed above section!

**Pull request merged**